WP_Auth0.php

```diff
@@ -3,13 +3,13 @@
  * Plugin Name: Login by Auth0
  * Plugin URL: https://auth0.com/docs/cms/wordpress
  * Description: Login by Auth0 provides improved username/password login, passwordless login, social login, multi-factor authentication, and single sign-on for all your sites.
- * Version: 4.3.1
+ * Version: 4.3.2
  * Author: Auth0
  * Author URI: https://auth0.com
  * Text Domain: wp-auth0
  */
 
-define( 'WPA0_VERSION', '4.3.1' );
+define( 'WPA0_VERSION', '4.3.2' );
```

Comment:
If this PR is bumping the version for a next release, it should be done in a separate PR.

Reply:
Whoops, good catch! That was in there for debugging purposes, forgot to remove from the commit. Fixing!